decidim-core/spec/system/notifications_spec.rb

```diff
@@ -112,6 +112,31 @@
     end
   end
 
+  context "with comment notifications" do
```

Comment:
I think that we can drop this block as it should be already covered by `decidim-comments/spec/events/decidim/comments/comment_created_event_spec.rb`. In the case of the `context "with user group mentioned notifications"` block from #8598 it's OK because the missing group variable wasn't detected by its spec as I mentioned in [the comment review](https://github.com/decidim/decidim/pull/8598/files#r769728461).

Or maybe I'm missing something, and we should have a specific system spec for this case. If that's it, then please let me know 😄

Reply:
I realised it could be redundant, but since one is integration and the other unitary, I decided to keep both. Anyway, I agree it's better to have tests simple, so I'll delete this piece of code